db/version_set.cc

```diff
@@ -1358,9 +1359,11 @@ VersionSet::UpdatePenalty(
             {   // light penalty
                 count=(int)(static_cast<double>(level_bytes) / gLevelTraits[level].m_DesiredBytesForLevel);
                 value=count; // this approximates the number of compactions needed, no other penalty
+                value/=10;  // allow 10 files, approx 2G without throttle
```

Comment:
Current goal is one "value" for each 10 files of backlog.  Number of files is also an approximation based upon size.  Purely a trial and error value.